### PR TITLE
fix: add visitedNFT to localstorage

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] fix e2e test case
- [x] re-add visitedNFTs to localstorage

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

fix this error on localhost. how to reproduce: open gallery item in new tab

Screenshot 2023-03-13 at 20 27 01

### CODE REVIEW

File: stores/history.ts [hunk 1]
```diff
@@ -113,6 +113,11 @@ export const useHistoryStore = defineStore('history', {
         }
         this.visitedNFTs.unshift(historyItem)
       }
+
+      localStorage.setItem(
+        'history',
+        JSON.stringify({ history: { visitedNFTs: this.visitedNFTs } })
+      )
```
Comment (roiLeo) [inline stores/history.ts:120]: `useStorage()` might help (same stuff done in #5217)

**Review by roiLeo — COMMENTED** [review]
Didn't knew we used vuex-persist..
`preferences` still present over here will try to remove it

**Review by vikiival — APPROVED** [review]
